utils/releasetools/02_upload_tarball.sh

```diff
@@ -6,18 +6,18 @@ then
 fi
 
 echo "Uploading..."
-scp /tmp/redis-${1}.tar.gz ubuntu@host.redis.io:/var/www/download/releases/
+scp /tmp/valkey-${1}.tar.gz ubuntu@host.redis.io:/var/www/download/releases/
```

Comment:
Not sure it's worth immediately updating these scripts, since these scripts will also need to get updated since we aren't uploading to redis.io any more. @roshkhatri  What are your thoughts about what to do with these scripts?

Reply:
These can change in place. I am currently repurposing these scripts for the workflows.